driver/csiplugin/connectors/connectors.go

```diff
@@ -108,6 +108,7 @@ const (
 	UserSpecifiedCompression      string = "compression"
 	UserSpecifiedTier             string = "tier"
 	UserSpecifiedSnapWindow       string = "snapWindow"
```

Comment:
snapWindow is volumesnapshotclass parameter, not storageclass. Is there any specific reason it is here. Sorry though this change is not related to the PR.

Reply:
It is just a constant defined here along with other input parameter keys, snapWindow is used for create snapshot only.